cypress/support/commands.js

```diff
@@ -78,17 +80,14 @@ Cypress.Commands.add(
         },
       });
     }
-    if (contentType === 'Document' || contentType === 'News Item') {
+    if (['Document', 'News Item', 'CMSFolder'].includes(contentType)) {
```

Comment:
@ksuess nice improvement. Where does "CMSFolder" come from?

Reply:
CMSFolder is for Guillotina tests